core/tasks.py

```diff
@@ -0,0 +1,82 @@
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.mail import send_mail
+from django.shortcuts import get_object_or_404
+
+from core.models.country_programme import CPComment, CPReport
+from multilateralfund.celery import app
+
+logger = get_task_logger(__name__)
+User = get_user_model()
+
+
+@app.task()
+def send_mail_comment_submit(cp_comment_id):
+    cp_comment = get_object_or_404(CPComment, id=cp_comment_id)
+    cp_report = cp_comment.country_programme_report
+    link = f"{settings.FRONTEND_HOST[0]}/country-programme/{cp_report.country.iso3}/{cp_report.year}"
+    comment_section = CPComment.CPCommentSection(cp_comment.section).label
+
+    if cp_comment.comment_type == CPComment.CPCommentType.COMMENT_COUNTRY:
+        recipients = User.objects.filter(user_type=User.UserType.SECRETARIAT)
+    else:
+        recipients = User.objects.filter(
+            user_type=User.UserType.COUNTRY_USER, country=cp_report.country
+        )
+
+    for recipient in recipients:
+        send_mail(
```

Comment:
Erm you could send the mail to multiple user, don't think a `for` loop is needed.

Reply:
Yes... The only difference is that all the recipients will be visible in the `To` field.